src/legacy/core_plugins/timelion/public/timelion_vis_fn.ts

```diff
@@ -20,10 +20,34 @@
 import { get } from 'lodash';
 import { i18n } from '@kbn/i18n';
 import chrome from 'ui/chrome';
+import {
+  ExpressionFunction,
+  KibanaContext,
+  Render,
+} from 'src/plugins/data/common/expressions/types';
 import { TimelionRequestHandlerProvider } from './vis/timelion_request_handler';
 
-export const timelionVis = () => ({
-  name: 'timelion_vis',
+const name = 'timelion_vis';
+
+type Context = KibanaContext | null;
+
+interface Arguments {
+  expression: string;
+  interval: any;
+}
+
+export type VisParams = Required<Arguments>;
```

Comment:
Does `VisParams` need to be exported? If not let's remove that.

also (small nit): `Required<>` isn't really doing anything here; could just be `Arguments`

Reply:
Nice catch! Indeed, it shouldn't be exported. Removed